Mantra-UI/component/navbar/navbar.css

```diff
@@ -26,10 +26,42 @@
 }
 
 .mantra-nav-search {
-    margin: 0 30rem 0 40rem;
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    margin-left: auto;
+    justify-content: center;
 }
 
 .mantra-nav-footer {
+    display: flex;
+    align-items: center;
     margin-left: auto;
     margin-right: 6rem;
+    justify-content: space-between;
+}
+
+.mantra-login-btn {
+    margin-right: 10px;
+}
+
+@media only screen and (max-width: 480px) {
```

Comment:
below 610px of width, the content first card is going outside the box. please check

Reply:
Resolved